tests/test_sql_query.py

```diff
@@ -17,7 +17,6 @@
 #
 
 import os.path
-import asyncpg
```

Comment:
You could also maybe do
```
try:
    import asyncpg
except ImportError:
    pass
```
and add a comment saying that the tests will be skipped during setup if asyncpg isn't present.

Not sure if that's better.